pom.xml

```diff
@@ -112,8 +112,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
One interesting thing I've noticed.  You can set your source/target to 1.7, but compile against a 1.8 JDK to leverage the newer APIs.  As a result, you create 1.7 compatible binaries, but leverage 1.8 APIs (e.g. Stream).  No runtime issues unless a user tries to call one of those APIs.

It's an old trick, but finally figured out how it was working.